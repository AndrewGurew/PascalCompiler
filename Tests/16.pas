str:='A + B * c - 3'