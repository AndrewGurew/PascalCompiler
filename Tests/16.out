Position            Type                Text                Value               
--------------------------------------------------------------------------------
(1,1)               ID                  str                 str                 
(1,4)               Special symbol      :=                  :=                  
(1,6)               String              'A + B * c - 3'     A + B * c - 3       
